extensions/amp-form/amp-form.md

```diff
@@ -137,9 +137,7 @@ In the following example, the responses are rendered in an inline template insid
     </template>
   </div>
   <div submit-error>
-    <template type="amp-mustache">
-      Oops! {{name}}, {{message}}.
-    </template>
+    <template type="amp-mustache"> Oops! {{name}}, {{message}}. </template>
```

Comment:
Looks to me like this one autoindented incorrectly.

Reply:
This template block is now unwrapped into one single line. It's merely being displayed as wrapped by GitHub's UI.